examples/GetValueHttp/GetValueHttp.cpp

```diff
@@ -10,10 +11,13 @@
  * Define Constants
  ****************************************/
 
-#ifndef UBIDOTS_TOKEN
-#define UBIDOTS_TOKEN "...."  // Put here your Ubidots TOKEN
+#ifndef TOKEN
```

Comment:
This must be UBIDOTS_TOKEN  to avoid re-definitions of other instances in other libraries